packages/dataviews/src/components/dataviews-view-config/index.tsx

```diff
@@ -251,32 +253,42 @@ function ItemsPerPageControl() {
 	);
 }
 
-function FieldItem( {
-	field,
+function BaseFieldItem( {
+	fieldId,
+	label,
+	subLabel,
 	isVisible,
 	isFirst,
 	isLast,
 	canMove = true,
+	canHide = true,
+	isInteracting = false,
 	onToggleVisibility,
 	onMoveUp,
 	onMoveDown,
+	additionalActions,
```

Comment:
I'm wondering if it would be better to be able to pass `children` instead of naming it `additionalActions`.

Reply:
Good point I changed it to children.